YOLOv8SegROSNode:
  ros__parameters:
    # YOLO模型参数
    model_path: "/home/sunrise/Ebike_Human_Follower/src/yolov8_segment/models/yolov8n_instance_seg_bayese_640x640_nv12_modified3.0.bin"
    conf_threshold: 0.3
    kpt_conf_threshold: 0.3

    # 处理性能参数
    max_processing_fps: 15    
    
    # ok检测参数
    ok_confirm_frames: 10     # 连续3帧举手开始跟踪目标
    
    # 跟踪保护参数
    tracking_protection_time: 5.0
    
    # ReID相关参数
    reid_similarity_threshold: 0.73    # reid特征匹配阈值
    height_change_threshold: 0.28  # 新增高度变化阈值 
    lost_timeout_threshold: 10.0  # 最长丢失时间限制，超过则取消跟踪
    reid_model_path: "/home/sunrise/Ebike_Human_Follower/src/yolov8_detect_tracking/models/osnet_64x128_nv12.bin"
    
